outlines/fsm/json_schema.py

```diff
@@ -324,7 +339,7 @@ def to_regex(
                     {"type": "number"},
                     {"type": "boolean"},
                     {"type": "null"}
-                    # { "type": "array" },  # TODO: enable arrays within object-types
+                    # { "type": "array" },  # issue #913
```

Comment:
Is it possible to add a test that fails once the issue is addressed?

Reply:
It's easier to actually just fix the issue. It's a one character change plus tests.

Reply:
Should be good to remove the comment now